### ISSUE TITLE
Error in apt-get install -y for required packages

### ISSUE THREAD

**Issue by gawbul** [gawbul/docker-sge#4]
Getting the following error:

```
Processing triggers for libgdk-pixbuf2.0-0:amd64 (2.30.7-0ubuntu1) ...
Errors were encountered while processing:
 dictionaries-common
 aspell
 aspell-en
 hunspell-en-us
 libenchant1c2a:amd64
 libwebkitgtk-3.0-0:amd64
 enchant
 libyelp0
 yelp
 gnome-user-guide
 zenity
 unity-control-center
 unity-control-center-signon
 indicator-bluetooth
E: Sub-process /usr/bin/dpkg returned an error code (1)
2014/12/12 15:45:17 The command [/bin/sh -c apt-get update && apt-get -y install wget darcs git mercurial tcsh build-essential automake autoconf openssl libssl-dev munge libmunge2 libmunge-dev libjemalloc1 libjemalloc-dev db5.3-util libdb-dev libncurses5 libncurses5-dev libpam0g libpam0g-dev libpacklib-lesstif1-dev libmotif-dev libxmu-dev libxpm-dev hwloc libhwloc-dev openjdk-7-jre openjdk-7-jdk ant ant-optional javacc junit libswing-layout-java libxft2 libxft-dev libreadline-dev vim tmux man gawk] returned a non-zero code: 100
```

**Comment by gawbul** [comment]
Seems libhwloc-dev is requiring dependencies that install `unity*`, `gnome*` and `xserver-xorg*` packages. Could pin specific packages in this error list but might be better to stop all `unity*`, `gnome*` and `xserver-xorg*` packages installing?

**Comment by gawbul** [comment]
Fixed this in b0071112f49bcd8624dd9ea1dd3c618610a8379e.
